app/components/thumbnail_component.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class ThumbnailComponent < ApplicationComponent
+  attr_reader :resource
+
+  # @param [SolrDocument, WorkVersion] resource
+  def initialize(resource:)
+    @resource = resource
+  end
+
+  def icon
+    icon_map.fetch(resource.work_type, 'bar_chart')
+  end
+
+  def html_classes
+    if featured?
+      'thumbnail col-xxl-6 ft-work__img'
+    else
+      'thumbnail'
+    end
+  end
+
+  private
+
+    def featured?
+      current_page?('/')
```

Comment:
I'm not crazy about linking a dependency here between this component and the globally available helpers in Rails. For example, we might want featured displays somewhere else besides the homepage eventually (like maybe on a person's dashboard we might want to show "your most popular works" with a featured style).

how about we pass `featured` into the constructor as a keyword option that defaults to false?

Also as part of my navbar PR, I added a method, `ApplicationHelper::home?` that can do the `current_page?('/')` bit.